docker-compose.dev.yml

```diff
@@ -28,7 +28,8 @@ services:
       CVAT_DEBUG_WAIT: '${CVAT_DEBUG_WAIT_CLIENT:-no}'
       COVERAGE_PROCESS_START:
     ports:
-      - '127.0.0.1:9090:9090'
+      - "127.0.0.1:7000:8080"
```

Comment:
This is incorrect; in development port 7000 is supposed to be used by a server started by the developer that runs outside of the container.